src/sinks/kafka.rs

```diff
@@ -482,12 +482,15 @@ mod integration_test {
         consumer.assign(&tpl).unwrap();
 
         // wait for messages to show up
-        wait_for(|| {
-            let (_low, high) = consumer
-                .fetch_watermarks(&topic, 0, Duration::from_secs(3))
-                .unwrap();
-            future::ready(high > 0)
-        })
+        wait_for(
+            || {
+                let (_low, high) = consumer
+                    .fetch_watermarks(&topic, 0, Duration::from_secs(3))
+                    .unwrap();
+                future::ready(high > 0)
+            },
+            5,
+        )
```

Comment:
I'd prefer that we try to avoid changing unrelated code for the time being, just to help focus the PR.